torch/onnx/symbolic_opset12.py

```diff
@@ -12,17 +12,26 @@
 
 # This file exports ONNX ops for opset 12
 
+def einsum_helper(g, equation, tensors):
+    if not tensors:
+        raise RuntimeError("Einsum inputs are empty.")
+    if tensors[0].type().scalarType() == "Bool":
```

Comment:
is this because onnxruntime does not support Bool for einsum? If so please comment it out too.

Reply:
done